app/main.py

```diff
@@ -1,8 +1,55 @@
 class Car:
-    # write your code here
+    def __init__(self, comfort_class: int, clean_mark: int, brand: str):
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
+
     pass
+cars = [Car]
```

Comment:
The variable `cars` is initialized as a list containing the `Car` class itself, not instances of `Car`. You might want to create instances of `Car` and add them to the list instead.